rcon/rcon.py

```diff
@@ -105,9 +107,8 @@ class Rcon(ServerCtl):
     player_info_regexp = re.compile(r"(.*)\(((Allies)|(Axis))/(\d+)\)")
     log_time_regexp = re.compile(r".*\((\d+)\).*")
     connect_disconnect_pattern = re.compile(r"(.+) \((\d+)\)")
-    # Checking for steam ID length so people can't exploit it with a name like: short(Axis/123) ->
     kill_teamkill_pattern = re.compile(
-        r"(.*)\((?:Allies|Axis)\/(\d{17})\) -> (.*)\((?:Allies|Axis)\/(\d{17})\) with (.*)"
+        r"(.*)\((?:Allies|Axis)\/(.*)\) -> (.*)\((?:Allies|Axis)\/(.*)\) with (.*)"
```

Comment:
a bit greedy, we would probably be able to have discrete parts for either a numeric steam ID or a UUID (which seems to be the Windows Store ID). However, as we have no _guarantee_ that this is actually always a UUID, and given that the other possibilities of the whole line are expressed already, being greedy here is most likely no issue.

Scratch all what I wrote, I just was thinking out loud 😅